src/lib/optimizer/strategy/join_predicate_ordering_rule.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <memory>
+
+#include "abstract_rule.hpp"
+#include "logical_query_plan/abstract_lqp_node.hpp"
+
+namespace opossum {
+
+/**
+ * A rule that brings predicates in a multi-predicate-join into a (supposedly) efficient order.
```

Comment:
Elaborate? How is one order more efficient than another one?

Reply:
Is it clear now?